docs/_data/home/benefits.yml

```diff
@@ -0,0 +1,11 @@
+- heading: Lower your messaging costs
+  description: |
+    Pay less for messages with out competitive rates and never pay any service or integration fees.
```

Comment:
"without" rather than "with out"

Reply:
Good spot. This is supposed to with our*. I think I need to go through the wording a bit more with a comb.

Reply:
Even better spot, without reverses the meaning 😆

Reply:
Updated to be our*